index.html

```diff
@@ -255,7 +255,7 @@ <h1 class="white-header">FOSSASIA on social media</h1>
                         <!-- Ignore LineLengthBear -->
                         <div class="container">
                             <div class="row">
-                                <div class="col-xs-12 col-md-12" style="margin-bottom:20px;">
+                                <div class="col-xs-12 col-md-6 col-md-offset-3 yt">
```

Comment:
Yes, it would be automatically in the middle in the larger screen. The youtube section was not that much appealing when it is occupying 12grids on PC.